src/ucs/datastruct/khash.h

```diff
@@ -404,6 +404,8 @@ static const double __ac_HASH_UPPER = 0.77;
 #define KHASH_IMPL(name, khkey_t, khval_t, kh_is_map, __hash_func, __hash_equal) \
 	__KHASH_IMPL(name, static kh_inline klib_unused, khkey_t, khval_t, kh_is_map, __hash_func, __hash_equal)
 
+#define KHASH_STATIC_INITIALIZER {0}
```

Comment:
add ucs gtest:
khash_init_inplace and KHASH_STATIC_INITIALIZER give same result

Reply:
added gtest for hash init